mkt/api/tests/test_views.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf8 -*-
```

Comment:
mind changing this to `utf-8`? python doesn't care but emacs doesn't understand it without the hyphen